bunsen/bunsen-core/src/main/java/com/cerner/bunsen/definitions/EnumConverter.java

```diff
@@ -20,12 +20,13 @@ public void toHapi(Object input, IPrimitiveType primitive) {
     // We can't assume the type is necessarily `String` for Avro `string`
     // types; it can be Utf8 too, see:
     // https://github.com/apache/parquet-mr/commit/918609f2cc4e4de95445ce4fdd7dc952b9625017
-    primitive.setValueAsString(input.toString());
+
+    primitive.setValueAsString(input != null ? input.toString() : null);
```

Comment:
Hmm, shouldn't `toHapi` be annotated as `@Nullable`? Doesn't NullAway complain about this?